src/components/Card/docs/index.story.tsx

```diff
@@ -106,5 +63,89 @@ export default {
     media: mediaExamples,
     ratio: Object.keys(CardRatioOptions).map(key => CardRatioOptions[key]),
   },
-  examples: <Examples />,
+  dataHook: 'storybook-Card',
+  sections: [
+    header({
+      sourceUrl:
+        'https://github.com/wix/wix-ui-tpa/tree/master/src/components/Card',
```

Comment:
I don't think this is needed, it should be automatically added by reading the `.storybook/webpack.config.js` config